docs/ref/contrib/admin/index.txt

```diff
@@ -906,13 +910,17 @@ subclass::
 
                   def lookups(self, request, model_admin):
                       if request.user.is_superuser:
-                          return super(AuthDecadeBornListFilter,
-                              self).lookups(request, model_admin)
+                          return super(
+                              AuthDecadeBornListFilter,
+                              self,
+                          ).lookups(request, model_admin)
 
                   def queryset(self, request, queryset):
                       if request.user.is_superuser:
-                          return super(AuthDecadeBornListFilter,
-                              self).queryset(request, queryset)
+                          return super(
```

Comment:
A single line, with the .lookups is 100 chars in the docs.
The original did NOT have this on a single line, I just applied the style (as I understand it) to the code.